adminSDK/alertcenter/quickstart/src/main/java/AdminSDKAlertCenterQuickstart.java

```diff
@@ -36,12 +37,6 @@
 
   private static final String APPLICATION_NAME = "Google Admin SDK Alert Center API Java Quickstart";
   private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
-  /**
```

Comment:
Please keep the previous `SCOPES` variable and comment, as it contains important information about how to handle scope changes.